internal/controller/http/command.go

```diff
@@ -83,12 +83,17 @@ func (c *RestController) SetCommand(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	err = application.SetCommand(ctx, deviceName, commandName, queryParams, requestParamsMap, c.dic)
+	event, err := application.SetCommand(ctx, deviceName, commandName, queryParams, requestParamsMap, c.dic)
 	if err != nil {
 		c.sendEdgexError(w, r, err, common.ApiDeviceNameCommandNameRoute)
 		return
 	}
 
+	if event != nil {
+		correlationId := utils.FromContext(ctx, common.CorrelationHeader)
+		go sdkCommon.SendEvent(event, correlationId, c.dic)
```

Comment:
check reserved[common.PushEvent] and reserved[common.ReturnEvent] to decide how to response the result

Reply:
I misunderstood the requrement, resolving this change request